celesta-maven-plugin/src/test/java/ru/curs/celesta/plugin/maven/GenScoreResourcesMojoTest.java

```diff
@@ -52,8 +54,8 @@ public void testExecuteGenScoreResources() throws Exception {
         mojo.execute();
 
         List<String> grainPaths = Arrays.asList(
-            "data/table/table.sql",
-            "seq/sequence.sql"
```

Comment:
This code should be fine as well, and the variant with Stream.of is less readable IMHO.